src/test/java/neculaclaudiu/tests/seleniumpart1/TestPrimulSite.java

```diff
@@ -0,0 +1,117 @@
+package neculaclaudiu.tests.seleniumpart1;
+
+import neculaclaudiu.driver.ChromeDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.List;
+
+public class TestPrimulSite {
+    public static void main(String[] args) {
+        ChromeDriver driver = ChromeDriverManager.createChromeDriver();
+        testPrimulLink(driver);
+        //testAlDoileaLink(driver);
```

Comment:
Please don't leave these methods commented as we want to have functional code in main branch. Make the necessary changes to run one after another without failing. Thanks!